config/static.c

```diff
@@ -18,7 +18,6 @@
 
 #include <stdio.h>
 #include <stdbool.h>
-#include <lua.h>
```

Comment:
Once `set_lua_path()` is no longer exported, you won't need to remove this include.